pkg/config/setup/config.go

```diff
@@ -707,6 +707,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.reinvocation_policy", "IfNeeded")
 	config.BindEnvAndSetDefault("admission_controller.add_aks_selectors", false) // adds in the webhook some selectors that are required in AKS
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.enabled", true)
+	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.v2.enabled", false)
```

Comment:
IMHO, I think it would be better to have something like:

`config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.version","v1")`

This is more extensible in case we have other versions in the future.